guides/line-management/121s.md

```diff
@@ -18,22 +18,27 @@ You might talk about:
 
 
 ## Our 121 tool
-We use a tool called Small Improvements to schedule and take notes of all our 121's. These notes become data points that your direct reports can use to prepare for annual reviews and promotions.
-A scheduled 121 will have a list of 'Talking points' (agenda) that you can setup in advance.
-It's also an easy way for direct reports to ask peers for feedback and set objectives for career progression. 
+We use a tool called Small Improvements to schedule and take notes of all our 121's. These notes become data points that your direct reports can use to prepare for annual reviews and promotions. A scheduled 121 will have a list of 'Talking points' (agenda) that you can setup in advance. It's also an easy way for direct reports to ask peers for feedback and set objectives for career progression. 
 
 
 ## Line Manager 121 expectations
+
 We expect you to:
-- spend a minumum of 1 hour per month with each direct report
+
+- spend a minimum of 1 hour per month with each direct report
 - record 121s and notes on Small Improvements
+- to start with, set the agenda and lead the 121 meetings
 - provide clarity on role expectations and performance
-- help prepare direct reports work torwards career progression (or passing probation)
+- help prepare direct reports work towards career progression (or passing probation)
 - share and encourage feedback
 
+
 ## Direct report 121 expectations
-To start with its likely that you will set the agenda and lead the 121 meetings. Over time we want to build your direct reports confidence and empower them to take the lead. 
```

Comment:
I split this across the expectations of the manager and the report as there's a bit for both of them.